CHANGELOG.md

```diff
@@ -24,7 +24,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [0.150.1] - 2023-12-29
 ### Added
-- New flag `--get-token` for `kamu system api-server` cli command which additionaly prints
+- New flag `--get-token` for `kamu system api-server` cli command which additionally prints
```

Comment:
Thanks a lot for all the spelling fixes!

I already installed a spell checker, and mb you can add an entry into `DEVELOPER.md` to recommend installing one.

I'm using `Code Spell Checked` in VSCode, but found it too noisy in Rust files, so had to disable it completely for code, which is not great ... I'd like it to at least check the comments.

What setup are you using?

Reply:
> I already installed a spell checker, and mb you can add an entry into DEVELOPER.md to recommend installing one.

Unfortunately, I don't have a good recommendation here this time, as I'm just using the bundled in RustRover :(

> I'm using Code Spell Checked in VSCode, but found it too noisy in Rust files, so had to disable it completely for code, which is not great ... I'd like it to at least check the comments.

yes, I see -- I used to do that, too.

> What setup are you using?

For spellchecking: RustRover + DeepL + Grammarly 

---

PS. 

> This gave me a bit of a school flashback about getting all my spelling errors exposed by the teacher :)
> Thanks for fixing our dyslexic codebase!

I wish I had the level of English to be able to read such errors/descriptions just with my eyes -- compensate, at the moment, with the tools listed above :)